cfy_manager/components/prometheus/prometheus.py

```diff
@@ -0,0 +1,357 @@
+PROMETHEUS_VERSION = '2.18.1'
+NODE_EXPORTER_VERSION = '1.0.0'
+BLACKBOX_EXPORTER_VERSION = '0.16.0'
+POSTGRES_EXPORTER_VERSION = '0.8.0'
+RABBITMQ_EXPORTER_VERSION = '1.0.0-RC7'
+PROMETHEUS_DATA_DIR = join(sep, 'var', 'lib', 'prometheus')
+
+logger = get_logger(PROMETHEUS)
+
+
+class Prometheus(BaseComponent):
+    component_name = 'prometheus'
+
+    def install(self):
+        _install_prometheus()
+        _install_node_exporter()
+        _install_blackbox_exporter()
+        _install_postgres_exporter()
+        _install_rabbitmq_exporter()
+
+    def configure(self):
+        logger.notice('Configuring Prometheus Service...')
+        _deploy_configuration()
+        service.configure(PROMETHEUS, append_prefix=False)
+        service.configure(NODE_EXPORTER, append_prefix=False)
+        service.configure(BLACKBOX_EXPORTER, append_prefix=False)
+        service.configure(POSTGRES_EXPORTER, append_prefix=False)
+        service.configure(RABBITMQ_EXPORTER, append_prefix=False)
+        logger.notice('Prometheus successfully configured')
+
+    def remove(self):
+        logger.notice('Removing Prometheus and exporters...')
+        files.remove_files([
+            PROMETHEUS_DATA_DIR,
+            PROMETHEUS_CONFIG_DIR,
+        ], ignore_failure=True)
+        files.remove_files([join(BIN_DIR, file_name) for file_name in
+                            ('prometheus', 'promtool',
+                             NODE_EXPORTER,
+                             BLACKBOX_EXPORTER,
+                             POSTGRES_EXPORTER,
+                             RABBITMQ_EXPORTER,)],
+                           ignore_failure=True)
+        service.remove(NODE_EXPORTER, append_prefix=False)
+        service.remove(BLACKBOX_EXPORTER, append_prefix=False)
+        service.remove(POSTGRES_EXPORTER, append_prefix=False)
+        service.remove(RABBITMQ_EXPORTER, append_prefix=False)
+        service.remove(PROMETHEUS, append_prefix=False)
+        logger.notice('Successfully removed Prometheus and exporters files')
+
+    def start(self):
+        logger.notice('Starting Prometheus and exporters...')
+        service.restart(PROMETHEUS, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS]['port'])
+        service.restart(NODE_EXPORTER, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS][NODE_EXPORTER]['metrics_port'])
+        service.restart(BLACKBOX_EXPORTER, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS][BLACKBOX_EXPORTER]['metrics_port'])
+        service.restart(POSTGRES_EXPORTER, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS][POSTGRES_EXPORTER]['metrics_port'])
+        service.restart(RABBITMQ_EXPORTER, append_prefix=False,
+                        ignore_failure=True)
```

Comment:
@mateumann would not be cleaner to add all the required services as list and restart them like this
```
for target_service in target_services:
    service.restart(target_service, append_prefix=False, ignore_failure=True)
```

It will reduce the amount of restart and it will be more readable

Reply:
yeah, this one is much cleaner now as well :)